globals/Level.gd

```diff
@@ -0,0 +1,26 @@
+extends Node
+
+var current_xp: int = 0
+var current_level: int = 1
+const XP_PER_SECOND: int = 5
+var xp_to_next_level: int = 100
+
+func add_xp() -> void:
+	current_xp += XP_PER_SECOND
+	print("current_xp: ", current_xp)
+	print("current_level: ", current_level)
+	print("xp_to_next_levelXP: ", xp_to_next_level)
```

Comment:
Not sure if it was intentional but you left prints here

Reply:
I'm still working on this sorry